ui/admin/app/routes/scopes/scope/session-recordings/session-recording/channels-by-connection.js

```diff
@@ -50,20 +48,4 @@ export default class ScopesScopeSessionRecordingsSessionRecordingChannelsByConne
       storageBucket,
     };
   }
-
-  /**
-   * Reapplies storage policy dates to session recording
-   * @param {SessionRecordingModel}
-   */
-  @action
-  @notifyError(({ message }) => message, { catch: true })
-  @notifySuccess(({ end_time }) =>
```

Comment:
moved this to the root `session-recording` because we need the model to refresh to get the updated session recordings in the list view after the error happens on `reapply`